### ISSUE TITLE
Adapt this script without the need for vrp

### ISSUE THREAD

**Issue by DaxTter77** [George-Adam/vrp_searchtrash#1]
I will can adapt this script to use it without the need for vrp?

**Comment by George-Adam** [comment]
Yes, you can do that. Just remove the lib directory and the tunnel connections at the top of both client and server side. I do not think you need server side at all for the standalone version.
